sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -24,7 +24,7 @@
 
 logger = logging.getLogger(__file__)
 
-DATABASE_EXISTS_QUERY = 'select name, collation_name from sys.databases;'
+DATABASE_EXISTS_QUERY = "select name, collation_name from sys.databases;"
```

Comment:
Are all the those single quote -> double quote changes needed? Side-question because I'm not a python pro, what are the best practice/convention regarding python double-quotes/single-quotes?